scripts/docker_build/sof_builder/Dockerfile

```diff
@@ -66,9 +66,14 @@ ENV HOME /home/sof
 
 # build cross compiler
 USER sof
-RUN cd /home/sof && git clone https://github.com/thesofproject/xtensa-overlay.git && \
+RUN cd /home/sof && \
+	git clone \
+		--depth 1 --branch sof-gcc8.1 \
+		https://github.com/thesofproject/xtensa-overlay.git && \
 	cd xtensa-overlay && git checkout sof-gcc8.1 && cd ../ && \
-	git clone https://github.com/thesofproject/crosstool-ng.git && \
+	git clone \
+		--depth 1 --branch sof-gcc8.1 \
+		https://github.com/thesofproject/crosstool-ng.git && \
```

Comment:
Can you please remove the `git checkout` commands now that they are part of the `git clone`?